sklearn/model_selection/tests/test_split.py

```diff
@@ -425,24 +425,38 @@ def test_shuffle_kfold_stratifiedkfold_reproducibility():
 
     kf = KFold(3, shuffle=True, random_state=0)
     skf = StratifiedKFold(3, shuffle=True, random_state=0)
+    kf2 = KFold(3, shuffle=True, random_state=np.random.RandomState(0))
+    skf2 = StratifiedKFold(3, shuffle=True,
+                           random_state=np.random.RandomState(0))
 
-    for cv in (kf, skf):
+    for cv in (kf, skf, kf2, skf2):
```

Comment:
maybe add a comment saying "check that calling ``split`` twice yields the same results"